clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -91,7 +89,7 @@ func (a *apiResponsivenessGatherer) Gather(config *measurement.MeasurementConfig
 			if isBad {
 				prefix = "WARNING "
 			}
-			klog.Infof("%s: %vTop latency metric: %+v; threshold: %v", a, prefix, apiCall, sloThreshold)
+			klog.Infof("%s: %vTop latency metric: %+v; threshold: %v", apiResponsivenessMeasurementName, prefix, apiCall, sloThreshold)
```

Comment:
I prefer using `a` instead of `apiResponsivenessMeasurementName` and add `String()` method to `apiResponsivenessGatherer`. This will allow us to have name for a single instance of measurement. Currently instance name is same as measurement name, however I can imagine we want to change it in the future.